HousingRepairsOnlineApi/Properties/launchSettings.json

```diff
@@ -36,9 +36,15 @@
         "AZURE_STORAGE_CONNECTION_STRING": "<REPLACE WITH AZURE STORAGE CONNECTION STRING>",
         "STORAGE_CONTAINER_NAME": "housing-repairs-online",
         "GOV_NOTIFY_KEY" : "<REPLACE WITH GOV NOTIFY API URL>",
-        "CONFIRMATION_SMS_NOTIFY_TEMPLATE_ID": "<REPLACE WITH GOV NOTIFY CONFIRMATION SMS TEMPLATE ID>",
-        "CONFIRMATION_EMAIL_NOTIFY_TEMPLATE_ID": "<REPLACE WITH GOV NOTIFY CONFIRMATION EMAIL TEMPLATE ID>",
-        "INTERNAL_EMAIL_NOTIFY_TEMPLATE_ID": "<REPLACE WITH GOV NOTIFY INTERNAL EMAIL TEMPLATE ID>",
+        "TENANT_CONFIRMATION_SMS_NOTIFY_TEMPLATE_ID": "<REPLACE WITH GOV NOTIFY CONFIRMATION SMS TEMPLATE ID>",
```

Comment:
Changing the existing environment variable names and adding new ones will require a Terraform change too, which is not present on this PR.